runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -142,9 +144,21 @@ public long append(@Nonnull Set<UUID> streamIDs, @Nonnull Object object,
                 // If we're here, we succeeded, return the acquired token
                 return tokenResponse.getTokenValue();
             } catch (OverwriteException oe) {
+
+                if (retry-- == 0) {
+                    ILogData ld = new LogData(DataType.DATA, object);
+                    ILogData.SerializationHandle sh = ld.getSerializedForm();
+                    log.error("append[{}]: Overwrite , streams {}, write size {}",
```

Comment:
Should be more descriptive:  Overwritten after {} retries: would be more useful.